### PR TITLE
Milestone 1: Toolbar and Headline section

### PR DESCRIPTION
**Pull request by AbeeraTahir**
In this branch, I have added the toolbar and headline section of the mobile-first design of the Portfolio Project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <div class="primary_text">
+            Hey There. I’m Alex
+        </div>
+        <div class="supporting_text">
+            I’am a Software Developer
+        </div>
+        <div class="description white">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </div>
```
Comment (Olamarx) [inline index.html:33]: - [ ] Kindly make use of `p` tag here as this is a paragraph

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <div class="vector">
+                <img src="icons/angelist.png" alt="angelist">
+            </div>
+            <div class="vector">
+                <img src="icons/twitter.png" alt="vector">
+            </div>
+            <div class="vector">
+                <img src="icons/medium.png" alt="medium">
+            </div>
+        </div>
```
Comment (Olamarx) [inline index.html:50]: - [ ] Kindly use `ul` tag to wrap the whole links
- [ ] Kindly wrap each of the `img` in an `a` tag and add `href` to the `a` tag. learn more on how to use it [here](https://www.w3schools.com/tags/tag_a.asp)

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @AbeeraTahir,
 


Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## The Figma view. Don't mind the border around the paragraph
Uini

## Your project view 
gvft

Kindly make the design look very close to that on the figma.

## Highlights
- Good commit messages ✅ 
- Professional readme file ✅ 
- All test passed ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbeeraTahir** [comment]
Hi @Olamarx,

Thank you so much for your review of my project. I have done all the changes that you suggested to me in my index.html file, but I couldn't understand a few things in the review. These are:

1. What does it mean "don't mind the border around the paragraph"? Because I have not seen any border around the paragraph in the Figma design, and that is what I am a bit confused about.  I have added a border around the paragraph with the same color but I'll be very grateful to you If you please elaborate on this point further.

2. You said that "kindly make the design look very close to that on the Figma." I have now tried my best to make it look very close to the Figma design, but I'll be very grateful to you If you please tell me which parts of the design, now, I have to work to make it look very close to the Figma design.

Thank you.

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi @AbeeraTahir 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

 

## Highlights:
* Well written PR description ✔️ 
* Proper use of GitHub flow ✔️  
* All tests passed ✔️ 
* Great design, almost identical to the Figma design ✨ keep rocking 🚀
* Clean code ✨  

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**